importing w/ browserify

I'm running into an issue trying to import peaks.js into my project that builds with browserify.

I noticed that the browserify cdn is giving the same error:
http://wzrd.in/standalone/peaks.js

```
Error: "browserify exited with code 1"

code: 1
stderr: Error: Cannot find module './arraybuffer.js' from '/tmp/peaks.js116723-30618-kaoht5/node_modules/peaks.js'
    at /home/admin/browserify-cdn/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:55:21
    at load (/home/admin/browserify-cdn/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/home/admin/browserify-cdn/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /home/admin/browserify-cdn/node_modules/browserify/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:107:15)

dirPath: /tmp/peaks.js116723-30618-kaoht5

This is probably an issue with the package, and not browserify-cdn itself.
If you feel differently, feel free to file a bug report at:

    https://github.com/jfhbrook/browserify-cdn/issues

Include the ENTIRETY of the contents of this message, and the maintainer(s)
can try to help you out.

Have a nice day!
```

The issue seems to be that browserify is trying to resolve modules like `./arraybuffer.js` inside of peaks.js directory when it should be looking in the waveform-data directory. Does anyone have an idea for how to fix this?

--- a/package.json
+++ b/package.json
@@ -41,8 +41,8 @@
   "license": "LGPL-3.0",
   "scripts": {
     "build": "npm run build:main && npm run build:standalone",
-    "build:main": "browserify -e ./src/main.js -t deamdify -o index.js",
-    "build:standalone": "browserify -d -e ./src/main.js -t deamdify -s peaks | exorcist peaks.js.map > peaks.js",
+    "build:main": "browserify -e ./src/main.js -t deamdify | derequire - > index.js",
+    "build:standalone": "browserify -d -e ./src/main.js -t deamdify -s peaks | exorcist peaks.js.map | derequire - > peaks.js",
     "eslint": "eslint src/**/*.js test/**/*.js",
     "test": "./node_modules/karma/bin/karma start --single-run",
     "pretest": "jshint ./src",
@@ -56,6 +56,7 @@
     "browserify": "^13.0.0",
     "chai": "^3.0.0",
     "deamdify": "^0.2.0",
+    "derequire": "^2.0.3",
     "eslint": "^3.2.2",
     "exorcist": "^0.4.0",
     "github-changes": "^1.0.0",

